addon/models/fd-dev-uml-cod.js

```diff
@@ -1,9 +1,146 @@
+import Ember from 'ember';
 import { Model as DevUMLCODMixin, defineBaseModel  } from
   '../mixins/regenerated/models/fd-dev-uml-cod';
 import CODModel from './fd-cod';
 
+import FdUmlActiveObject from '../objects/uml-primitives/fd-uml-active-object';
+import FdUmlNAryAssociation from '../objects/uml-primitives/fd-uml-naryassociation';
+import FdUmlInstance from '../objects/uml-primitives/fd-uml-instance';
+import FdUmlMultiObject from '../objects/uml-primitives/fd-uml-multi-object';
+import FdUmlNote from '../objects/uml-primitives/fd-uml-note';
+import FdUmlGeneralization from '../objects/uml-primitives/fd-uml-generalization';
+import FdUmlNoteConnector from '../objects/uml-primitives/fd-uml-note-connector';
+import FdUmlQAggregation from '../objects/uml-primitives/fd-uml-qualified-aggregation';
+import FdUmlQComposition from '../objects/uml-primitives/fd-uml-qualified-composition';
+import FdUmlQAssociation from '../objects/uml-primitives/fd-uml-qualified-association';
+import FdUmlAssociation from '../objects/uml-primitives/fd-uml-association';
+import FdUmlAggregationLink from '../objects/uml-primitives/fd-uml-aggregation-link';
+import FdUmlAssociationLink from '../objects/uml-primitives/fd-uml-association-link';
+import FdUmlCompositionLink from '../objects/uml-primitives/fd-uml-composition-link';
+import FdUmlDesignPatternConnector from '../objects/uml-primitives/fd-uml-design-pattern-connector';
+import FdUmlDesignPattern from '../objects/uml-primitives/fd-uml-design-pattern';
+import FdUmlUsecaseActor from '../objects/uml-primitives/fd-uml-usecase-actor';
+import FdUmlCollFlatMsgForward from '../objects/uml-primitives/fd-uml-forward-flat-message';
+import FdUmlCollNestedMsgForward from '../objects/uml-primitives/fd-uml-forward-nested-message';
+import FdUmlCollAsyncMsgForward from '../objects/uml-primitives/fd-uml-forward-async-message';
+import FdUmlCollNestedMsgBack from '../objects/uml-primitives/fd-uml-backward-nested-message';
+import FdUmlCollFlatMsgBack from '../objects/uml-primitives/fd-uml-backward-flat-message';
+import FdUmlCollAsyncMsgBack from '../objects/uml-primitives/fd-uml-backward-async-message';
```

Comment:
What a zoo of names?! Please use the following rules:
* The class name for `JointJS` matches, as far as possible, with the existing one on the server.
* The class name of the `ember` object is the name of the class for `JointJS` with the prefix `fd-uml-`.
* The filename is the name of the class of the `ember` object in the `dasherize` syntax.